k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala

```diff
@@ -55,6 +56,21 @@ class EndpointsNamer(
       Activity.value(NameTree.Neg)
   }
 
+  override def getAllNames: Activity[Set[Path]] = {
```

Comment:
Wait -- doesn't this mean that that this activity can't actually behave properly if new namespaces are added?